### PR TITLE
Fix default controller

### PR DESCRIPTION
**Pull request by VasylPosmit**
defaultController still need specs.

### CODE REVIEW

File: src/app/common/default.controller.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+      self.user = userService.user;
+      self.data = sectionsService.data[self.state];
+      self.calculate = calculate;
+
+      activate();
+
+      function activate() {
+        self.closeSidenav();
+        if (_.isEqual(userService.user, userService.mockUser)) {
```
Comment (andrenovax) [inline src/app/common/default.controller.js:31]: what are you doing here?

Comment (VasylPosmit) [inline src/app/common/default.controller.js:31]: That is the way to defer app load from state change. 

Already changed in branch add-offline- mode https://github.com/VasylPosmit/health/blob/add-offline-mode/src/app/common/default.controller.js#L33

Comment (andrenovax) [inline src/app/common/default.controller.js:31]: why do you make timeout? can you remove it?

Comment (VasylPosmit) [inline src/app/common/default.controller.js:31]: I can remove it only with `if (_.isEqual(userService.user, userService.mockUser)).`

The reason is $http request [runs at the runblock ](https://github.com/VasylPosmit/health/blob/add-offline-mode/src/app/config.js#L42)asynchronously and return promise with userData.

I need to apply userData in controller (self.user = userService.user) **after** promise resolved. I count that $http request took 1300 ms so made timeout for this function.

Comment (andrenovax) [inline src/app/common/default.controller.js:31]: this is a hack, dude
if something should be done after,
it should be done after,
not after some period.

run does not work for your case and actually it is for app settings only
you can get user in controller;)

Comment (VasylPosmit) [inline src/app/common/default.controller.js:31]: [It's the best solution ](https://github.com/VasylPosmit/health/pull/26/commits/f896199d9b109efb36deeb6824e44f3f4f2e5cfe#diff-ba3a8e9b2091a4198eaeca2956c0e0c7R32)of about 10 that I had implemented 

It's about ok.

**Pull request merged**